pom.xml

```diff
@@ -120,14 +215,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
why are we removing this?

Reply:
The declaration isn't needed, unit tests are run by default using this plugin.

`docker run --rm -v $(pwd):/workspace -w /workspace maven:3-eclipse-temurin-17 mvn clean package`

```
[INFO] Compiling 2 source files to /workspace/target/test-classes
[INFO]
[INFO] --- surefire:3.0.0:test (default-test) @ akash-java ---
```